packages/web/src/pages/oauth-login-page/OAuthLoginPage.tsx

```diff
@@ -353,11 +351,13 @@ export const OAuthLoginPage = () => {
                     [styles.permissionTextExtraLight]: !userEmail
                   })}
                 >
-                  {userEmail || (
+                  {userEmail == null ? (
```

Comment:
Is there a case where `userEmail` is undefined? Should we be using `===` here

Reply:
I always use == null by convention, but === would work here too!